packages/main/src/ColorPicker.ts

```diff
@@ -111,6 +112,16 @@ class ColorPicker extends UI5Element implements IFormInputElement {
 	@property()
 	name?: string;
 
+	/**
+	 * Defines the display mode of the component.
+	 * @default "Default"
+	 * @since 2.5.0
+	 *
```

Comment:
redunant empty row, do it after the description, before the `@default` tag maybe